### PR TITLE
Add solution

### PR DESCRIPTION
**Pull request by pavlokhomiak**

### CODE REVIEW

**Review by fyarkiy — APPROVED** [review]
Good work!

File: pom.xml [hunk 1]
```diff
@@ -8,5 +8,99 @@
...
 
+    <dependencies>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-war-plugin</artifactId>
```
Comment (Sofasmile) [inline pom.xml]: do we need war plugin here

File: pom.xml [hunk 2]
```diff
@@ -8,5 +8,99 @@
...
+                    <encoding>UTF-8</encoding>
+                    <consoleOutput>true</consoleOutput>
+                    <failsOnError>true</failsOnError>
+                    <linkXRef>false</linkXRef>
+                </configuration>
+
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
```
Comment (Sofasmile) [inline pom.xml]: the same)

File: src/main/java/threads/MyRunnable.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package threads;
+
+import main.Counter;
+import org.apache.log4j.Logger;
+
+public class MyRunnable implements Runnable {
+    private static final Logger logger = Logger.getLogger(MyRunnable.class);
+    private static final int MAX_INT = 100;
```
Comment (Sofasmile) [inline src/main/java/threads/MyRunnable.java]: we have two same constants. let's move this value to Counter class and create a method getMaxNumber() or smth else

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**